core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -77,6 +76,12 @@ class SparkEnv (
 
   def shuffleManager: ShuffleManager = _shuffleManager
 
+  // We initialize the MemoryManager later in SparkContext after DriverPlugin is loaded
+  // to allow the plugin to overwrite memory configurations
```

Comment:
nit I assume this is overwriting executor memory configurations please update comment

Reply:
updated